infrastructures/kubernetes/src/test/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/pvc/CommonPVCStrategyTest.java

```diff
@@ -279,8 +294,15 @@ public void shouldDeletePVCsIfThereIsNoPersistAttributeInWorkspaceConfigWhenClea
       throws Exception {
     // given
     Workspace workspace = mock(Workspace.class);
+    Page workspaces = mock(Page.class);
+
     lenient().when(workspace.getId()).thenReturn(WORKSPACE_ID);
 
+    lenient()
```

Comment:
why do you add `lenient()`  inside the test?

Reply:
could you clarify a bit, seems to be smth that is used all over the shop

Reply:
lenient is a workaround. That sometimes used in setup methods. It means that this stubbing might be unused in some tests. I see no reason to add useless stubbing inside the test. Or maybe it not useless?  In that case, I do not need lenient.